src/pages/ResultsListPage.js

```diff
@@ -8,6 +8,8 @@ import mui from "material-ui";
 
 import BaseResultsPage from "./BaseResultsPage";
 import ResultListItem from '../components/ResultListItem';
+import CrisisLineItem from '../components/CrisisLineItem';
+import HeaderBar from '../components/HeaderBar';
```

Comment:
Already available as `components.` also I can't see where you're using them. Merge artifact?